osu.Game/Overlays/Changelog/ChangelogBuild.cs

```diff
@@ -69,34 +69,69 @@ public ChangelogBuild(APIChangelogBuild build)
                         Margin = new MarginPadding { Vertical = 5 },
                     };
 
+                    var entryColor = entry.Major != null && (bool)entry.Major ? OsuColour.FromHex("#fd5") : Color4.White;
```

Comment:
please use the colour from the palette, not hardcoded locally.

Reply:
the variable could be renamed to `entryColour` to stay consistent with the language